README.md

```diff
@@ -10,6 +10,20 @@ In addition, the migration files can be written in Groovy in order to run script
 Only requirement is that migration files follow the pattern `V99__Some_name.[sql|groovy]`, where 
 `99` can be any integer value.
 
+### Patch and override files
+
+If you have broken migration scripts, you can use the migration tool to add patch and override files to existing migration files. These two files are always created together.
+
+1. If the broken file has already been applied, a patch file will be used. The patch file name should follow this pattern:
+   `V99_file_to_fix.patch.[sql|groovy]`
+
+2. If the broken file hasn't been applied, an override file will be used instead of the original file. The override file name should follow this pattern:
+   `V99_file_to_fix.override.[sql|groovy]`
+
```

Comment:
Better, but I think we need to add a clear description of the behavior because it's essential to avoid ambiguity. This snippet taken from this ticket should be enough

```
WHEN file.sql exists and was executed

AND file.override.sql exists -> nothing happens
AND file.patch.sql exists -> apply patch
WHEN file.sql exists and was not executed

AND file.override.sql exists -> apply override
AND file.patch.sql exists -> apply file.sql and apply patch
AND file.patch.sql exists AND file.override.sql exists -> apply override
WHEN file.sql doesn't exist but file.patch.sql exists OR file.override.sql exists -> nothing happens

```

Reply:
Even tho I'm not understanding the different in the last two statements 

```
AND file.patch.sql exists -> apply file.sql and apply patch
AND file.patch.sql exists AND file.override.sql exists -> apply override
```

Reply:
Both `.patch` and `.override` [must exist](https://github.com/seqeralabs/migtool/blob/c55425d82e7a28b98093214dbe4bd31bd01b93c9/src/main/java/io/seqera/migtool/MigTool.java#L319-L321) together, so I would describe the cases in this way:

```
WHEN file.sql exists and was executed
AND file.patch.sql, file.override.sql pair exists -> apply patch

WHEN file.sql exists and was not executed
AND file.patch.sql, file.override.sql pair exists -> apply override

WHEN file.sql exists
AND only one of file.patch.sql or file.override.sql exists -> illegal state, the migration process is aborted before any migration files are applied
```

Reply:
so many words for a tool used only by us IMHO

Reply:
``` For each migration file that needs to be corrected a pair of .path and .override must exists. The behaviour will follow these rules:

- **WHEN** file.sql exists and was executed **AND** file.patch.sql, file.override.sql pair exists -> apply patch
- **WHEN** file.sql exists and was not executed **AND** file.patch.sql, file.override.sql pair exists -> apply override

If for a given file there is only either a single .patch or a single .override then the tool will throw an illegal state exception stopping the migration process ```

Reply:
ok, let's update the readme

Reply:
@weronikasosnowskaseqera please...also be sure to change the tower-cloud PR with the files